src/amber/router/pipe/reload.cr

```diff
@@ -0,0 +1,26 @@
+require "../../support/client_reload"
+
+module Amber
+  module Pipe
+    # Reload clients browsers using `ClientReload`.
+    #
+    # NOTE: Amber::Pipe::Reload is intended for use in a development environment.
+    # ```
+    # pipeline :web do
+    #   plug Amber::Pipe::Reload.new
+    # end
+    # ```
+    class Reload < Base
+      def initialize
+        Support::ClientReload.new
+        super
+      end
+
+      def call(context : HTTP::Server::Context)
+        context.response.print Support::ClientReload::INJECTED_CODE
```

Comment:
Does print close the response?

Reply:
@eliasjpr I don't think so

What other method should I use?

Reply:
If it does not close the response thats good ccause `Pipe::Last` has to run.